controlsfx/src/main/java/org/controlsfx/validation/ValidationMessage.java

```diff
@@ -84,6 +84,35 @@ public static ValidationMessage error( Control target, String text ) {
     public static ValidationMessage warning( Control target, String text ) {
         return new SimpleValidationMessage(target, text, Severity.WARNING);
     }
+    
+    /**
+     * Factory method to create a simple info message 
+     * @param target message target
+     * @param text message text 
+     * @return info message
+     */
+    public static ValidationMessage info( Control target , String text ) {
+    	return new SimpleValidationMessage(target, text, Severity.INFO);
+    }
+
+    /**
+     * Factory method to create a simple ok message 
+     * @param target message target
+     * @param text message text 
+     * @return ok message
+     */
+    public static ValidationMessage ok( Control target , String text ) {
+    	return new SimpleValidationMessage(target, text, Severity.OK);
+    }
+
+    /**
+     * Factory method to create a simple ok message 
+     * @param target message target
+     * @return ok message
+     */
+    public static ValidationMessage ok( Control target ) {
```

Comment:
When would a user use this method? There is no message while the documentation says its supposed to show a message.

Reply:
See answer in the follow up question...